### ISSUE TITLE
Puedo decir que un conjunto A es numerable si demuestro que es equinumeroso con los naturales?

### ISSUE THREAD

**Issue by dvillas28** [IIC1253/IIC1253-2024-1#171]
Tal vez sea media obvia la pregunta, pero en la definición de conjunto numerable postula una igualdad entre cardinalidades. Mi pregunta es si me puedo pegar ese 'salto' desde el concepto de equinumerabililad entre conjuntos, al concepto de conjunto numerable?

Saludos! 🤠

**Comment by pibahamondesw** [comment]
Sí, de hecho nosotros lo definimos así en clases (un conjunto enumerable es aquel con misma cardinalidad que los naturales, que es lo mismo que lo que preguntas).
